### ISSUE TITLE
Tax and inventory toggles are not working

### ISSUE THREAD

**Issue by ajporlante** [reactioncommerce/reaction#4401]
## Prerequisites
* [X] Are you running the latest version?
* [X] Are you able to consistently reproduce the issue?
* [X] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
In product detail form,  the toggle switches in tax and inventory sections cannot be toggled.

## Steps to Reproduce
1. Edit a product variant.
2. Try to toggle the switches in the tax and inventory sections. From the UI, the switches will not "react."
3. Refresh the page and you will see that the switches were actually toggled.

## Possible Solution
Update the components/autoform

### Versions
```
Node: 8.11.2
NPM: 5.6.0
Meteor Node: 8.11.2
Meteor NPM: 5.10.0
Reaction CLI: 0.29.0
Reaction: 1.14.0
Reaction branch: 1.14.0
```

**Comment by ajporlante** [comment]
Hmm. I just assumed that the toggle switches did nothing because I'm unable to toggle them. But looking at the product collection, these switches correspond to some fields and should be fixed to work instead of hiding them.

I'm revising this issue.

**Comment by zenweasel** [comment]
This does not appear to be fixed

**Comment by spencern** [comment]
Closed by #4445
